docs/_templates/components_requirements.tmpl

```diff
@@ -1,39 +1,76 @@
 .. -*- mode: rst -*-
 
-Inmanta core and extensions versions
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The orchestrator is constituted of several components (python packages and npm packages).
-The python packages are of two types: one core package and several extension packages.
 
-{% if data['component_requirements']%}
 
-The following table shows the ranges of the Inmanta Python components, this product version is compatible with.
-The lower bound of these version constraints will remain the same for the whole lifetime of this major version. Upper bounds may be added when a new major version is released.
+{% if data['module_compatibility_ranges']%}
 
-This table is only meant as a reference for developers of Inmanta extensions and is meant to be used as input to a
-dependency resolver e.g. pip to ensure the resulting versions are compatible.
-This information is not relevant when installing the orchestrator via RPM as is recommended
-in the :ref:`install-server` section.
 
-.. list-table:: Inmanta core and extensions versions
-   :header-rows: 1
+Module compatibility ranges
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   * - .. centered:: Component
-     - .. centered:: Required version
 
-   {% for key, value in data["component_requirements"] | dictsort %}
-   * - .. centered:: {{ key }}
-     - .. centered:: {{ value }}
-   {% endfor %}
+During module development, when the version of the module integrates tightly with a specific ISO version,
+we need a way to tell the CI which version of the module should be used per ISO line. This can be achieved by either:
 
-.. note::
+- Using a specific index per ISO version
```

Comment:
I think I know what you mean here, namely that we have indexes/git repos that only contain the versions for a specific ISO version. But I think that an end-user will need a more elaborate explanation to be able to understand this.